app/assets/json/translations/react/es.json

```diff
@@ -468,10 +598,11 @@
   "label.doneAddingPeople": "Ya he agregado a todas las personas",
   "label.doneWithThisAddress": "Ya he finalizado con esta dirección",
   "label.dontWantPreference": "No deseo esta preferencia para el sorteo",
+  "label.downloadApplication": "Descargar solicitud en %{lang}",
```

Comment:
content mismatch with the English translation file (applies to other non-english translation files)